src/components/admin/MainIssueController.jsx

```diff
@@ -156,8 +156,26 @@ export default class MainIssueController extends FalcorController {
               return false;
             }
           }
+          if (/<a.*?href\s*?=\s*?["'](?!http).*?>/.test(article.html)) {
+            if (!window.confirm(
+              `${article.title} is in non-absolute format,
```

Comment:
Template strings (when you're using the backticks, which you were doing correctly for the `article.title` interpolation!) include newlines and whitespace, so this string will actually be printed as
```
SOME_TITLE is in non-absolute format,
              which means that it does not have http(s):// in front of it. 
```